samba/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 9.0
+
+- New option `veto_files` to limit writing of specified files to the share. This is a breaking change, please add "veto_files": [] to your config.
```

Comment:
Because we have a default, they will be merged into the current config. You can remove that need